src/wallet/UserWalletTransactions.js

```diff
@@ -1,114 +1,169 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
 import { FormattedNumber } from 'react-intl';
+import ReduxInfiniteScroll from 'redux-infinite-scroll';
+import { defaultAccountLimit } from '../helpers/apiHelpers';
+import Loading from '../components/Icon/Loading';
 import ReceiveTransaction from './ReceiveTransaction';
 import TransferTransaction from './TransferTransaction';
 import SavingsTransaction from './SavingsTransaction';
 import PowerUpTransaction from './PowerUpTransaction';
 import ClaimReward from './ClaimReward';
 import './UserWalletTransactions.less';
 
-const getFormattedTransactionAmount = (amount, currency) => {
-  const transaction = amount.split(' ');
-  const transactionAmount = parseFloat(transaction[0]).toFixed(3);
-  const transactionCurrency = currency || transaction[1];
-  return (
-    <span>
-      <FormattedNumber
-        value={transactionAmount}
-        minimumFractionDigits={3}
-        maximumFractionDigits={3}
-      />
-      {` ${transactionCurrency}`}
-    </span>
-  );
-};
+class UserWalletTransactions extends Component {
```

Comment:
I think we use `React.Component` everywhere instead of importing `Component` directly.

Reply:
yeah we do, will update now